agents-api/agents/bookmark_importer/routes.py

```
@@ -2,267 +2,283 @@
         for category_name in request.selected_categories:
             if category_name in available_categories:
                 selected_categories.append(available_categories[category_name])
-        
+
         if not selected_categories:
-            raise HTTPException(
-                status_code=400,
-                detail="No valid categories selected"
-            )
-        
+            raise HTTPException(status_code=400, detail="No valid categories selected")
```

Ruff (0.13.3)

Abstract `raise` to an inner function

(TRY301)